lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,186 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"__main__.py", "qutebrowser"}}
```

Comment:
You're right. I bet I get `__main__.py` because I'm running the qutebrowser from the repository instead of using out of date debian packages. I'm changing the class line to the one you suggested.